tests/e2e/functional_test.go

```diff
@@ -372,6 +373,87 @@ var _ = Describe("Functional e2e", Serial, func() {
 
 	time.Sleep(2 * time.Second)
 
+	// TODO: maybe move this into a ppnd-specific test suite?
+	// Can include tests for allowing data loss which involve all 3 pause requesters (isbsvc, numaflow controller, and pipeline itself)
```

Comment:
added [issue](https://github.com/numaproj/numaplane/issues/509) to move this to a PPND-specific test